### PR TITLE
Feature/mortar 52

### PR DESCRIPTION
**Pull request by vamidjo**
https://jira.nationalgeographic.com/browse/MORTAR-52

Testing Instructions: 

1. git checkout feature/mortar-52
2. gulp serve 
3. visit any page (e.g., http://localhost:3000)

### CODE REVIEW

**Comment by xiwcx** [comment]
looks good! let's get #34 resolved first so we can reference it's new icon classes in this footer.

**Comment by xiwcx** [comment]
OK, if you merge release/2.0 in to this branch then you see the github icon in the docs. Then add one of the new icon classes on to your icon instead of the inline styles and we'll get this merged.

File: app/site/_layouts/default.html [hunk 1]
```diff
@@ -35,6 +35,30 @@
...
+
+        <div class="mt_subh4">
+          <a href="http://www.nationalgeographic.com/community/privacy/">Privacy Policy</a> |
+          <a href="http://www.nationalgeographic.com/community/terms/">Terms of Service </a>
+        </div>
+      </div>
+
+      <div class="mt_col-lg-3">
+        <div class="mt_subh4">
+          <svg class="mt_icon--large" style="vertical-align: bottom;">
```
Comment (xiwcx) [inline app/site/_layouts/default.html]: i'm gonna go ahead and fix it myself, but now that we have shame.scss let's stick all of our random styles there so we don't lose track of them.

Comment (xiwcx) [inline app/site/_layouts/default.html]: and if you run across any, remove them and stick them in shame.

Comment (vamidjo) [inline app/site/_layouts/default.html]: @xiwcx all of the inline styles?

**Pull request merged**